### PR TITLE
update to 0.1.3

### PR DESCRIPTION
**Pull request by cbouss**
Changes:
- Update to 0.1.3

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,39 +1,44 @@
...
   name: appnope
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/a/appnope/appnope-{{ version }}.tar.gz
-  sha256: dd83cd4b5b460958838f6eb3000c660b1f9caf2a5b1de4264e941512f603258a
+  sha256: 02bd91c4de869fbb1e1c50aafc4098827a7a54ab2f39d9dcba6c9547ed920e24
 
 build:
   number: 1001
```
Comment (danpetry) [inline recipe/meta.yaml]: what's this for?

File: recipe/meta.yaml [hunk 2]
```diff
@@ -1,39 +1,44 @@
...
 test:
   imports:
     - appnope
+  requires:
+    - pip
+  commands:
+    - pip check
 
 about:
   home: http://github.com/minrk/appnope
```
Comment (danpetry) [inline recipe/meta.yaml]:
```suggestion
  home: https://github.com/minrk/appnope
```

File: recipe/meta.yaml [hunk 3]
```diff
@@ -1,39 +1,44 @@
...
-  license: BSD 2-Clause
+  license: BSD-2-Clause
   license_family: BSD
   license_file: LICENSE
   summary: Disable App Nap on OS X 10.9
   description: |
     appnope is a simple package for disabling App Nap on OS X 10.9, which can be problematic.
   doc_url: https://pypi.python.org/pypi/appnope/0.1.0
```
Comment (danpetry) [inline recipe/meta.yaml]:
```suggestion
  doc_url: https://github.com/minrk/appnope/blob/main/README.md
```

File: recipe/meta.yaml (hunk 1 above)
Comment (cbouss) [reply recipe/meta.yaml]: One package depends on it on osx. 0.1.2 imports from distutils, which as been removed as of 3.12.

Comment (danpetry) [reply recipe/meta.yaml]: sorry, I meant the build number being 1001

**Review by JeanChristopheMorinPerso — APPROVED** [review]

File: recipe/meta.yaml [hunk 4]
```diff
@@ -1,39 +1,44 @@
...
 source:
   url: https://pypi.io/packages/source/a/appnope/appnope-{{ version }}.tar.gz
-  sha256: dd83cd4b5b460958838f6eb3000c660b1f9caf2a5b1de4264e941512f603258a
+  sha256: 02bd91c4de869fbb1e1c50aafc4098827a7a54ab2f39d9dcba6c9547ed920e24
 
 build:
   number: 1001
   skip: true  # [not osx]
-  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv"
+  script: "{{ PYTHON }} -m pip install . --no-deps --no-build-isolation --ignore-installed --no-cache-dir -vvv"
```
Comment (skupr-anaconda) [inline recipe/meta.yaml]:
```suggestion
  script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation --ignore-installed --no-cache-dir -vv
```

File: recipe/meta.yaml (hunk 1 above)
Comment (skupr-anaconda) [reply recipe/meta.yaml]: It was made by conda-forge 5 years ago https://github.com/conda-forge/appnope-feedstock/pull/4/files but the reason is unknown. I would reset it to `0` as conda-forge did https://github.com/conda-forge/appnope-feedstock/blob/main/recipe/meta.yaml#L12

File: recipe/meta.yaml (hunk 4 above)
Comment (cbouss) [reply recipe/meta.yaml]: addressed

File: recipe/meta.yaml (hunk 1 above)
Comment (cbouss) [reply recipe/meta.yaml]: addressed

**Review by danpetry — APPROVED** [review]

**Pull request merged**